src/screens/ChatInfoScreen/ChatInfoScreen.tsx

```diff
@@ -196,6 +200,9 @@ const ChatInfoScreen = () => {
     loadingChannelInfo,
     isLoadingInitChat
   } = useChatInfoScreenHook();
+  const navigation = useNavigation();
+
+  const {onLeaveGroup, onReportGroup} = useGroupInfo(channelInfo?.id);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `useGroupInfo` hook is used to get the functions `onLeaveGroup` and `onReportGroup`. These functions are necessary for the new features of leaving a group chat and reporting a group chat.
```diff
- const {signedProfileId} = useUserAuthHook();
+ const navigation = useNavigation();
+ const {onLeaveGroup, onReportGroup} = useGroupInfo(channelInfo?.id);
+ const {signedProfileId} = useUserAuthHook();
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->